react/javascript/demo-external-api/README.md

```diff
@@ -21,32 +21,32 @@ This demo requires a project with access to the sheets API and an OAUTH2 client
 8. Click `Save and continue` on the `Edit app registration - Scopes` page.
 9. If you selected the `External User type` you will need to enter Test users on the`Edit app registration - Test users` page. Click `Save and Continue`.
 10. Click the `Credentials` link in the sidebar.
-11. Click the `Create Credentials` button and select `OAUTH client ID`.
+11. Click the `Create Credentials` button and select `OAuth client ID`.
 12. Select `Application Type` of `Web Application`.
 13. Enter a `Name` for your application.
 14. Add the base url of the Looker instance that will host your extension as an `Authorized JavaScript Origin`. Example `https://myinstance.looker.com`.
-15. Add the base url of the Looker instance that will host your extension appended with `/extensions/oauth2_redirect` as an `Authorized redirect URI`. Example `https://myinstance.looker.com/extensions/oauth2_redirect`.
+15. Add the base url of the Looker instance that will host your extension appended with `/extensions/oauth_redirect` as an `Authorized redirect URI`. Example `https://myinstance.looker.com/extensions/oauth_redirect`.
```

Comment:
like that url name change

Reply:
Let me double check that! Thats what I get for doing global change!